packs/lamp/config.yml

```diff
@@ -11,7 +11,7 @@ description: |+
   This LAMP observability pack allows you to monitor a LAMP stack (Linux, Apache, MySQL, PHP).
   It includes a specific dashboard to get observability for an entire LAMP stack.
 
-  NOTE: Observability pack references are currently not yet implemented,
+  Observability pack references are currently not yet implemented,
```

Comment:
Also update this to quickstarts? or is that part of another PR to move the name?

Reply:
I fixed this and also removed the line about installing other quickstarts as I think this is covered now.